Packs/CommonScripts/Scripts/ClearUserSession/ClearUserSession.py

```diff
@@ -0,0 +1,421 @@
+                    "Message": "",
+                }
+                if user_name not in users_ids:
+                    user_output["Result"] = "Failed"
+                    user_output["Message"] = "Username not found or no integration configured."
+                    outputs[user_name] = user_output
+                    continue
+
+                success: bool = False
+                brands_succeeded: list = []
+                brands_failed: list = []
+                failed_message = ""
+
+                # Okta v2
+                if okta_v2_id := get_user_id(users_ids, OKTA_BRAND, user_name):
+                    okta_clear_user_sessions_command = Command(
+                        name="okta-clear-user-sessions",
+                        args={"userId": okta_v2_id},
+                        brand=OKTA_BRAND,
+                    )
+                    if okta_clear_user_sessions_command.is_valid_args():
+                        readable_outputs, _, error_message = clear_user_sessions(okta_clear_user_sessions_command)
+                        results_for_verbose.extend(readable_outputs)
+                        if not error_message:
+                            success = True
+                            brands_succeeded.append(OKTA_BRAND)
+                        else:
+                            failed_message += f"Okta v2: {error_message.lstrip('#').strip()}"
+                            demisto.debug(f"Failed to clear sessions for Okta user with ID {okta_v2_id}. "
+                                          f"Error message: {error_message}. Response details: {readable_outputs}.")
+                            brands_failed.append(OKTA_BRAND)
+
+                # Microsoft Graph User
+                if microsoft_graph_id := get_user_id(users_ids, brand_name=MS_GRAPH_BRAND, user_name=user_name):
+                    msgraph_user_session_revoke_command = Command(
+                        name="msgraph-user-session-revoke",
+                        args={"user": microsoft_graph_id},
+                        brand=MS_GRAPH_BRAND,
+                    )
+                    if msgraph_user_session_revoke_command.is_valid_args():
+                        readable_outputs, human_readable, _ = clear_user_sessions(msgraph_user_session_revoke_command)
+                        results_for_verbose.extend(readable_outputs)
+                        if "successfully" in human_readable:
+                            success = True
+                            brands_succeeded.append(MS_GRAPH_BRAND)
+                        else:
+                            brands_failed.append(MS_GRAPH_BRAND)
+                            failed_message += f"\nMG User: {human_readable.lstrip('#').strip()}"
+                            demisto.debug(f"Failed to clear sessions for Microsoft Graph user with ID {microsoft_graph_id}. "
+                                          f"Response details: {readable_outputs}")
+
+                if success:
```

Comment:
Why not just check that the list of the brand succeeded is not empty?

Reply:
Right, great idea.